src/PayPal/Order/CommandHandler/CapturePayPalOrderCommandHandler.php

```diff
@@ -103,45 +152,55 @@ public function handle(CapturePayPalOrderCommand $capturePayPalOrderCommand)
             $payload['vault'] = true;
         }
 
-        $response = $this->maaslandHttpClient->captureOrder($payload);
+        return $payload;
+    }
 
-        $orderPayPal = json_decode($response->getBody(), true);
+    private function savePrestaShopPayPalCustomerRelationship($vault)
+    {
+        try {
+            $payPalCustomerId = new PayPalCustomerId($vault['customer']['id']);
+            $customerId = new CustomerId($this->prestaShopContext->getCustomerId());
+            $this->payPalCustomerRepository->save($customerId, $payPalCustomerId);
+
+            return $vault;
+        } catch (CustomerException $exception) {
+            $this->logger->error($exception->getMessage());
+            return [];
+        } catch (\InvalidArgumentException $exception) {
+            return [];
+        } catch (\Exception $exception) {
+            $this->logger->error(sprintf('An error occurs during process : %s', $exception->getMessage()));
+            return [];
+        }
```

Comment:
If you decide to keep catch clauses here, then you don't need a return at each one. Just one at the end of the function